Base/home/anon/tests/run-tests-and-shutdown.sh

```diff
@@ -25,10 +31,10 @@ run(index) {
 #     UserEmulator: Tests designed to run inside the Userspace Emulator
 #     stack-smash: Intentionally crashes by smashing the stack
 #     TestJSON: AK/TestJSON makes assumptions about $PWD to load its input files
-#     .frm: Test inputs that are not tests
+#     .frm,.txt: Test inputs that are not tests
 #     test-web: Requires the window server in order to work
 #     test-js: We start this one manually with the show progress flag set to false
-exclude_patterns='Kernel/Legacy|.inc|UserEmulator|stack-smash|TestJSON|.frm|test-web|test-js'
+exclude_patterns='Kernel/Legacy|.inc|UserEmulator|stack-smash|TestJSON|.txt|.frm|test-web|test-js'
```

Comment:
now that the current working directory is set correctly, can we remove TestJSON from the exclude patterns?

Reply:
It's out of scope and it's taking already too long to merge this PR, can we do it in another PR?

Reply:
Sure.